src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -1613,7 +1612,7 @@ public void onSuccess(StreamState streamState)
                         }
                         progressSupport.progress("bootstrap", new ProgressEvent(ProgressEventType.COMPLETE, 1, 1, "Resume bootstrap complete"));
                         if (!isNativeTransportRunning())
```

Comment:
Should the condition be `!isNativeTransportRunning() || !isRPCServerRunning()`

Reply:
I didn't want to change this as it was unrelated to the bug.  Its also a specific feature about resuming bootstrap so feel its best to leave as is and only init if native isn't running.